src/main/java/me/zeroeightsix/kami/gui/rgui/component/ToggleButton.kt

```diff
@@ -0,0 +1,27 @@
+package me.zeroeightsix.kami.gui.rgui.component
+
+import me.zeroeightsix.kami.util.math.Vec2f
+
+class ToggleButton(
+    name: String,
+    description: String,
+    val check: () -> Boolean,
```

Comment:
shouldn't `check` be called `display`, as in display value? I understand in terms of "check button" but it might be less intuitive